gslib/gcs_json_api.py

```diff
@@ -2403,8 +2403,12 @@ def _TranslateApitoolsException(self,
         if 'The bucket you tried to delete was not empty.' in str(e):
           return NotEmptyException('BucketNotEmpty (%s)' % bucket_name,
                                    status=e.status_code)
-        return ServiceException('Bucket %s already exists.' % bucket_name,
-                                status=e.status_code)
+        return ServiceException(
+            'A Cloud Storage bucket named \'%s\' already exists. Try another '
```

Comment:
"Try another name" might not be necessary

Reply:
This text came from the UXers and I think they want that there so that there's a clear next step. I think I'll leave it as is for now.